build_rules.gradle

```diff
@@ -489,6 +491,15 @@ ext.applyJavaNature = {
     }
   }
 
+  // Enable errorprone, not by default right now
+  if (configuration.enableErrorProne) {
+    apply plugin: 'net.ltgt.errorprone'
+    dependencies {
+      errorprone 'com.google.guava:guava:22.0'
```

Comment:
When taking a look at the errorprone gradle plugin documentation, it says to just specify the line below to use a fixed version. Any reason why you also need to specify guava?

Reply:
Our version of Guava has incompatible classes. I can't recall if you get a method not found or a class not found if you omit this. It depends on the version of errorprone you choose. I wanted to prioritize recent errorprone for better checks, while our Guava is pinned way in the past.

Reply:
I will experiment with removing this - I may have added this first, and only later added the conditional for treating the `errorprone` configuration specially. Since it is an independent configuration, if we don't mess with it I would expect it to "just work".

Reply:
That would be great.

Reply:
Done. Seems to work.